components/webext-storage/src/migration.rs

```diff
@@ -198,7 +286,30 @@ fn do_insert(tx: &Transaction<'_>, ext_id: &str, vals: Vec<(String, Value)>) ->
             ":data": &Value::Object(map),
         },
     )?;
-    Ok(())
+    Ok(num_entries)
+}
+
+#[derive(Debug, Clone, Default, PartialEq)]
+pub struct MigrationInfo {
+    /// The number of entries (rows in the original table) we attempted to
+    /// migrate. Zero if there was some error in computing this number.
+    pub entries: usize,
+    /// The number of records we successfully migrated (equal to `entries` for
+    /// entirely successful migrations).
```

Comment:
Hmm, this implication of "different kinds of success" is making my Spidey-Sense tingle a little bit 🕷️ What does it mean if we attempted to migrate all entries, but `extensions` doesn't match that count?

Reply:
There's no more redundancy here.